manager/src/main/java/io/streamzi/openshift/API.java

```diff
@@ -82,16 +86,31 @@ public String getProcessorFlowDeployment(@PathParam("name") String name) {
     @Path("/processors")
     @Produces("application/json")
     public List<String> listProcessors() {
-        List<String> results = new ArrayList<>();
 
-        ConfigMapList configMapList = container.getOSClient().configMaps().inNamespace(container.getNamespace()).withLabel("streamzi.io/kind", "processor").list();
-        List<ConfigMap> processorConfigMaps = configMapList.getItems();
-        for (ConfigMap cm : processorConfigMaps) {
+
+        CustomResourceDefinition procCRD =  container.getOSClient().customResourceDefinitions().withName("processors.streamzi.io").get();
+        if(procCRD == null){
+            logger.severe("Can't find CRD");
+            return new ArrayList<>();
```

Comment:
If this can be immutable, and no elements need to be added, we could look at using `Collections.emptyList()`